SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+  We support the LTS Java releases and the current Java release.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability:
+  Under the repository name, click Security. If you cannot see the "Security" tab, 
```

Comment:
Line starts with an indentation that shouldnt be present. Also this line ends with a hidden space that should be removed.

Reply:
Fixing

Reply:
Looks good to me.